File: Java/Petshop.java
public class Petshop {
    
    private String ID;
    private String nama_product;
    private String kategori_product;
    private int Harga_product;

    Petshop(){
        this.ID = "";
        this.nama_product = "";
        this.kategori_product = "";
        this.Harga_product = 0;
    }

    Petshop(String ID, String nama_product, String kategori_product, int Harga_product){
        this.ID = ID;
        this.nama_product = nama_product;
        this.kategori_product = kategori_product;
        this.Harga_product = Harga_product;
    }

    public void setHarga_product(int harga_product) {
        Harga_product = harga_product;
    }

    public void setID(String iD) {
        ID = iD;
    }
    
    public void setKategori_product(String kategori_product) {
        this.kategori_product = kategori_product;
    }

    public void setNama_product(String nama_product) {
        this.nama_product = nama_product;
    }

    public String getID() {
        return ID;
    }

    public String getKategori_product() {
        return kategori_product;
    }

    public String getNama_product() {
        return nama_product;
    }

    public int getHarga_product() {
        return Harga_product;
    }

    public void Ingfo(){
        System.out.println("ID          : "+ ID );
        System.out.println("Nama        : "+ nama_product);
        System.out.println("Kategori    : "+ kategori_product);
        System.out.println("Harga       : "+ Harga_product);
        System.out.println("//=================//");
    }


}

File: Python/Petshop.py
class Petshop:

    def __init__(self, ID, Nama, Kategori, Harga):
        self.ID = ID
        self.Nama = Nama
        self.Kategori = Kategori
        self.Harga = Harga
        

    def setID(self, ID):
        self.ID = ID

    def setNama(self, Nama):
        self.Nama = Nama

    def setKategori(self, Kategori):
        self.Kategori = Kategori

    def setHarga(self, Harga):
        self.Harga = Harga

    def getID(self):
        return self.ID

    def getNama(self):
        return self.Nama

    def getKategori(self):
        return self.Kategori

    def getHarga(self):
        return self.Harga

    
    def Infoo():
        print("ID          : {ID}\n")
        print("Nama        : {Nama}\n"  )
        print("Kategori    : {Kategori}\n" )
        print("Harga       : {Harga}\n" )
